app/services/project-skills-list.js

```diff
@@ -0,0 +1,58 @@
+import Ember from 'ember';
+import skillsList from 'code-corps-ember/utils/skills-list';
+
+const {
+  computed: {
+    alias
+  },
+  get,
+  getProperties,
+  inject: { service },
+  Service,
+  set
+} = Ember;
+
+export default Service.extend({
```

Comment:
This seems to be almost the same as the `user-skills-list` service. It might be worth it to either create a mixin for these services or perhaps a base service that they extend.